lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1971,13 +1993,15 @@ def create_course(self):
             display_name='Empty',
         )
 
-    def test_grade_report(self):
+    def test_course_grade_report_for_all_learners(self):
```

Comment:
@DawoudSheraz The explanation given for `ProblemGradeReport` above, also applies here for `CourseGradeReport`.